src/eventhandlerdsp.cxx

```diff
@@ -309,7 +309,7 @@ void handleDspEvents()
 				if ( availableRead >= sizeof(EventTrackPan) ) {
 					EventTrackPan ev;
 					jack_ringbuffer_read( rbToDsp, (char*)&ev, sizeof(EventTrackPan) );
-					jack->getLogic()->trackPan( ev.track, ev.pan );
+					jack->getLogic()->trackPan( ev.track, ev.pan, ev.source );
 					//jack->bindingTrack = ev.track;
```

Comment:
This line needs uncommented and the pan bind will work :)

Reply:
Thanks, that totally makes sense.